### ISSUE TITLE
Tabs in scrollEnabled TabBar of fixed width not fully visible.

### ISSUE THREAD

**Issue by Palisand** [react-native-community/react-native-tab-view#503]
### Current behaviour

```javascript
<TabBar scrollEnabled />
```
works as expected:
![default](https://user-images.githubusercontent.com/4138177/38520964-4e39f964-3c12-11e8-9175-e578534284c5.gif)

**But adding a fixed width:**
```javascript
<TabBar scrollEnabled style={{width: 300}} />
```
**results in:**
![tabbar](https://user-images.githubusercontent.com/4138177/38520999-627c3888-3c12-11e8-9c1c-1d7fa61fabb6.gif)

And applying a width to `TabViewAnimated` will just lead to:
![tabview](https://user-images.githubusercontent.com/4138177/38521084-9cda1518-3c12-11e8-8af0-d054a6b021bb.gif)
as expected.

### Expected behaviour

```javascript
<TabBar scrollEnabled style={{width: 300}} />
```
Should result in a `TabBar` that behaves like the third gif, but with `Scenes` that extend to the entire container, as in the first and second gifs.

### Code sample

```javascript
import React, {Component} from 'react';
import {View, Text} from 'react-native';
import {TabViewAnimated, TabBar, SceneMap} from 'react-native-tab-view';

class ScrollEnabledTabView extends Component {
  state = {
    index: 0,
    routes: [
      {key: 'foo', title: 'Foo'},
      {key: 'bar', title: 'Bar'},
      {key: 'baz', title: 'Baz'},
      {key: 'qux', title: 'Qux'},
    ],
  };

  renderHeader = props => (
    <TabBar
      {...props}
      scrollEnabled
      style={{width: 300}}
      tabStyle={{padding: 0, width: 100}}
    />
  );
  
  renderScene = SceneMap({
    foo: () => <View><Text>Foo</Text></View>,
    bar: () => <View><Text>Bar</Text></View>,
    baz: () => <View><Text>Baz</Text></View>,
    qux: () => <View><Text>Qux</Text></View>,
  });

  handleIndexChange = index => this.setState({index});

  render() {
    return (
      <TabViewAnimated
        navigationState={this.state}
        renderHeader={this.renderHeader}
        renderScene={this.renderScene}
        onIndexChange={this.handeIndexChange}
        style={{flex: 1}}
      />
    );
  }
}
```

### What have you tried

There is a solution, but it's not that pretty:
```javascript
<TabBar
  ...props
  scrollEnabled
  style={{width: 300}}
  layout={{...props.layout, width: 300}} // Overwrite width of layout prop
/>
```
**This performs smoothly on iOS but leads to some jitters on Android.**

### Your Environment

| software              | version
| --------------------- | -------
| ios or android        | BOTH
| react-native          | 0.54.2
| react-native-tab-view | 0.0.77
| node                  | v8.4.0
| npm           | 5.7.1

**Comment by satya164** [comment]
Hey, I just released a new alpha `2.0.0-alpha.0` of the library. It's rewritten using [react-native-gesture-handler](https://github.com/kmagiera/react-native-gesture-handler) and [react-native-reanimated](https://github.com/kmagiera/react-native-reanimated) addresses a many platform specific bugs and performance problems. The documentation is updated as well.

Please try the new version and see if it addresses your issue. If not, please open a new issue following the issue template.

**Comment by renaekathleen** [comment]
@Palisand did upgrading to v2.0 fix your issue? I have the same issue.
